content/en/platform/corda/5.0/deploying-operating/observability/metrics.md

```diff
@@ -49,38 +51,166 @@ metrics:
-| `corda_uniqueness_checker_subbatch_size` | DistributionSummary | The number of requests in a sub-batch processed by the uniqueness checker. |
-| `corda_uniqueness_checker_request_count` | Counter | Cumulative number of requests processed by the uniqueness checker, irrespective of batch. |
-| `corda_uniqueness_backingstore_session_execution_time` | Timer | The overall execution time for a (uniqueness checker) backing store session, which includes retrieving uniqueness database connection details, getting a database connection, as well as all database operations (both read and write) carried out within a session context. |
-| `corda_uniqueness_backingstore_transaction_execution_time` | Timer | The execution time for a transaction within the context of a backing store session, which excludes retrieving uniqueness database connection details and getting a database connection. If a transaction needs to be retried due to database exceptions, then the execution time covers the cumulative duration of all retry attempts. |
-| `corda_uniqueness_backingstore_transaction_error_count` | Counter | Cumulative number of errors raised by the backing store when executing a transaction. This is incremented regardless of whether an expected or unexpected error is raised, and is incremented on each retry. |
-| `corda_uniqueness_backingstore_transaction_attempts` | DistributionSummary | The number of attempts that were made by the backing store before a transaction ultimately succeeded. In the event that a transaction was unsuccessful due to reaching the maximum number of attempts, this metric is not updated. |
-| `corda_uniqueness_backingstore_db_commit_time` | Timer | The time taken by the backing store to commit a transaction (that is, write) to the database. Only updated if data is written to the database, so it is not cumulative across retry attempts for a given transaction. |
-| `corda_uniqueness_backingstore_db_read_time` | Timer | The time taken by the backing store to perform a single read operation from the database. |
+The following Corda-specific metrics are exported and they have been added at the following levels:
+
+* HTTP requests
+* Sandbox
+* Messaging
+* Flow
+* Flow mapper
+* Flow session
+* Peer-to-peer messages and sessions
+* Ledger uniqueness checker client service
+* Uniqueness checker
+* Backing store
+
+#### HTTP Requests
+
+The REST server acts as a mediator, converting HTTP requests into messages that can be consumed by the Corda workers.
+Two metrics offer insights into the HTTP requests: the cumulative count of requests received over a specific duration,
+and the processing time for each request. There is a maximum time limit, or timeout, imposed on the processing of each
+HTTP request. If a timeout is reached, an error message is dispatched to the HTTP client.
+
+<style>
+table th:first-of-type {
+    width: 25%;
```

Comment:
```suggestion
    width: 25%;
```

I'd probably make this 35% or so so the first column doesn't need to break. Additionally, could narrow the second column and then make the third one wider.

Reply:
Reformatted the table